client/packages/programs/src/JsonForms/common/JsonForm.tsx

```diff
@@ -158,9 +162,21 @@ const FormComponent = ({
       return error.message ?? '';
     });
 
-  // This allows "default" values to be set in the JSON schema
+  // This allows "default" values to be set in the JSON schema, though it
+  // currently can't add defaults on nested properties unless a
+  // default object is defined at the root level -- see issue #6971
```

Comment:
Off track, but just expanded the comment here to clarify the limitations, as spelled out in #6971